backend/local/local.go

```diff
@@ -265,6 +167,10 @@ func (b *Local) Create() (backend.Blob, error) {
 
 // Construct path for given Type and name.
 func filename(base string, t backend.Type, name string) string {
+	if t == backend.Config {
```

Comment:
Because `dirname()` is also used to construct glob patterns for listing types and the config file isn't stored in a directory below the repository. So it's just a completely different case.